locations/jclouds/src/main/java/org/apache/brooklyn/location/jclouds/JcloudsLocation.java

```diff
@@ -1608,7 +1606,35 @@ protected SshMachineLocation createTemporarySshMachineLocation(HostAndPort hostA
         }
     }
 
-    
+    /**
+     * Creates a temporary WinRM machine location (i.e. will not be persisted), which uses the given credentials.
```

Comment:
Good catch. I'll add that as well. @sjcorbett Might explain why we see various extra persisted location files, as we do the same for temporary `SshMachineLocation` files!